app/controllers/data_requests_controller.rb

```diff
@@ -36,6 +37,9 @@ def create
         format.json { respond_with unscoped_exportable, data_request }
       end
     end
+  rescue ArgumentError
+    skip_authorization
```

Comment:
Why skip authorization here just to return the 422 response code? 

Shouldn't authorization run before / first thing on the create or if it's not needed have an exclusion on the end points it's not needed for? 

Looks like the operations handle authorization so why are they in the controller as well?
https://github.com/zooniverse/caesar/blob/97832c797551d90848468e880577bf2a61c27d81/app/controllers/data_requests_controller.rb#L32

Reply:
The comment means that the creation of the export requests will do the authorization for the export request specifically, but there are no corresponding operations for the other actions (ShowsDataRequests, ListsDataRequests, etc) so those authorizations are left to the controller. I did refactor this so that a single `skip_authorization` invocation covers the entire method